rfc/account-usage-get-capability.md

```diff
@@ -0,0 +1,221 @@
+# Account Usage Get
+
+![draft](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square)
+
+## Editors
+
+- Storacha team
+- Natalie Bravo
+
+## Introduction
+
+The `usage/report` capability enables an agent to obtain storage usage information on a space. 
+
+Currently, whenever a user wants to view their total storage usage, whether via CLI or console, we can only provide this information if the agent being used has permission to access every space. In practice, this often isn’t the case<sup>[1](#note1)</sup>, which leads to a misalignment between the current definition of this capability and how it’s actually used across our system.
+
+With Storacha’s growing adoption through other tools, such as the Telegram Mini App and Bluesky, the total usage view has become even more fragmented.
+
+This RFC proposes a discussion on better approaches.
+
+
+<a id="note1"></a>
+> <sub>**Note 1:** Scenarios where we don't have the total view of the users spaces: 
+> -  If a user has lost the private key for a space and didn’t delegate control to us.  
+> - If a user didn’t delegate control to us and is using a device without the appropriate delegation from a space. 
+> - If a user logs in to a new agent (B) but creates a new space (2) on the old agent (A), agent B wouldn’t know about space 2.</sub>
+
+
+## Proposal
+
+A new capability that can be invoked in the context of an Account DID (aggregating usage across all spaces). 
+
+### `account-usage/get`
+
+#### Invocation
+
+```ipldsch
+type AccountUsageGet struct {
+  with AccountDID
+  nb optional AccountUsageGetNB
+}
+
+type AccountUsageGetNB struct {
+  space optional [SpaceDID]
+
+  # Optional time period (Unix timestamps in seconds).
+  # If omitted, provider MUST return a current snapshot.
+  period optional Period
+}
+
+type Period struct {
+  from Int # inclusive
+  to Int # inclusive
+}
+```
+
+> example: getting the total usage
```

Comment:
What is the "total usage"? What is the period in the returned data in this case?

Reply:
updated the text with more context